recipe/meta.yaml

```diff
@@ -72,6 +72,7 @@ requirements:
     - xlrd >=1.2.0
     - xlsxwriter
     - {{ pin_compatible('numpy') }}
+    - filelock >=3.8.0  # remove when new canvas-core is released
```

Comment:
```suggestion
```
The new version of orange-canvas-core is already on conda-forge so it can be removed.

Reply:
Thanks. But then I'd have to change the canvas-core requirement and I would not want to do it in the same release.

Reply:
Makes sense.